src/components/statusHandler/StatusHandler.tsx

```diff
@@ -100,34 +100,38 @@ const StatusHandler = ({
           type="warning"
           content={
             <div aria-live="polite" role="alert" data-testid={testDataId}>
-              <h6>
-                ({successfulImports}) test case(s) were imported. The following
-                ({failedImports.length}) test case(s) could not be imported.
-                Please ensure that your formatting is correct and try again.
-              </h6>
-              <ul>
-                {failedImports.map((failedImport) => {
-                  const family = failedImport?.familyName;
-                  const given = failedImport?.givenNames?.toString();
-                  const names =
-                    family && given
-                      ? `${family} ${given}`
-                      : failedImport?.patientId;
-                  return (
-                    <li data-testid="failed-test-cases">
-                      {names} <br />
-                      <span tw="ml-4">Reason : {failedImport.message}</span>
-                    </li>
-                  );
-                })}
-              </ul>
+              {failedImports.length > 0 && (
```

Comment:
What about failedImports length is 0? Will there be success message?

Reply:
if nothings failed, I don't think we need status alert. it used to display something like this:

`(10) test case(s) were imported. The following (0) test case(s) could not be imported. Please ensure that your formatting is correct and try again.` that seems lil confusing. we get toast message anyways if import was successful.

Reply:
without that condition it looks like this:
![image](https://github.com/MeasureAuthoringTool/madie-patient/assets/23436747/b9119f9f-d85f-452b-9986-39d973ef6af0)

Reply:
@CeciliaLiu8 We made a few changes to display messages dynamically and also changed the format slightly